grafana-alloy/README.md

```diff
@@ -67,19 +73,39 @@ The topology is illustrated in this diagram.
 ```mermaid
 flowchart LR
   greptimedb[(GreptimeDB)]
-  alloy[Alloy]
   grafana[Grafana]
 
-  alloy --> greptimedb
+  subgraph alloy[Alloy]
+    prometheus_exporter_unix --> prometheus_relabel
+    prometheus_relabel --> prometheus_remote_write
+
+    prometheus_relabel --> otelcol_receiver_prometheus
+    otelcol_receiver_prometheus --> otelcol_processor_transform
+    otelcol_processor_transform --> otelcol_exporter_otlphttp
+  end
+
+  prometheus_remote_write --> |PRW| greptimedb
+  otelcol_exporter_otlphttp --> |OTLP HTTP/Protobuf| greptimedb
+
   greptimedb --> grafana
 ```
 
+Grafana Alloy is a telemetry data pipeline that ingests data in Prometheus,
+Loki, and OpenTelemetry formats. It also provides processing capabilities, such
+as Prometheus relabeling and OpenTelemetry OTTL (OpenTelemetry Transformation
+Language) functions.
+
+In this example, we generate metrics data from Alloy's built-in data source
+called `prometheus.exporter.unix` and export the data to both a Prometheus
+remote write sink and an OpenTelemetry-compatible collector, which is
+GreptimeDB.
+
 ## Run in GreptimeCloud
 
 By default, this example writes data into a GreptimeDB instance within the
 docker compose. It's also possible to write to your own GreptimeCloud instance
 by creating a `greptime.env` file from our sample `greptime.env.sample` and
```

Comment:
Added a link to greptime.env.sample, it looks like [`greptime.env.sample`](./greptime.env.sample) ?

Reply:
I think it's ok. The idea is to let user copy the sample and rename